idprovider/local/peer.go

```diff
@@ -14,12 +14,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package contactsyaml
+package local
 
 import "github.com/hyperledger-labs/perun-node"
 
-// PeerEqual returns true if all fields in the Peer except OffChainAddr are equal.
-func PeerEqual(p1, p2 perun.Peer) bool {
+// PeerEqual returns true if all fields in the Peer ID except OffChainAddr are equal.
+func PeerEqual(p1, p2 perun.PeerID) bool {
```

Comment:
```suggestion
func PeerIDEqual(p1, p2 perun.PeerID) bool {
```
As we renamed `Peer` to `PeerID`, renaming this function accordingly. Since this is an exported function, please update using refactor command for the change to be reflected in other places as well.

Reply:
done